### ISSUE TITLE
[Northrend] [Quests] The Black Knight's Orders

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3919]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 At the moment, it looks like the gryphon mount is walking alot of times, instead of properly flying and using the correct route to get to the destination. Minor glitch involving a path issue I imagine and smth else.

**Expected behaviour**: Tell us what should happen instead
 The gryphon should fly as soon as you take him and it should use a correct route, not walk/fly on the ground and in the air.
**Steps to reproduce the problem**:

1.  Take the quest.
2.  Use the mount.
3. 

**Include proofs for this behaviour**

 Only got a few screenshots and 1 of them depicts said problem. Wish I'd ps sooner/faster :P.
 https://postimg.org/gallery/2b5zeqygo/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
http://wotlk.openwow.com/quest=13663

**Comment by Rushor** [comment]
it's true that he should walk some time https://youtu.be/RJrwFyvhNbY?t=61

but he should fly all the way in the air above the sea

**Comment by MrAndersonX** [comment]
Yeah, he's partially bugged. Still, it's a good thing the quest works tho.

**Comment by Rushor** [comment]
DB/Quest: The Black Knight's Orders - Black Knight''s Gryphon should be able to fly above the sea 

* closes https://github.com/dalaranwow/dalaran-wow/issues/3919
* okay the Black Knight''s Gryphon will now also fly a bit on the ground, but the main problem, that the Black Knight''s Gryphon was not able to execute the flyanimation above the sea, was fixed now
* commit: 725ca2fb3ca2fc079c08388b2c33268247794a19

![GitHub Logo](http://i.imgur.com/6wtP5hx.jpg)
Black Knight''s Gryphon should be able to fly above the sea 
![GitHub Logo](http://i.imgur.com/nF3o13C.png)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
